### DISCUSSION TITLE
Repeatable field limit?

### DISCUSSION THREAD

**Discussion by semiplay** [Laravel-Backpack/community-forum#1058] [Q&A (Help)] [answered]
I have a client site with a large amount of data.  In one model I have a repeatable field which is for steps of instructions, and it is stored as json.  I have the step limit set to 100, but we just discovered that it stops at 52, which is sort of weird.  Any steps added past 52 don't save.  I confirmed that the new steps are being sent in the payload, just not saved.  

I manually entered the steps into the json in the database, and they showed in the model on the edit page - but if I save that page, it removes any steps after 52.  So it seems like the database field is ok with the information.  (I also tried making it a LONGTEXT field and had the same result.)

Also, when saving, all the items prior to 52 are updated fine, it just removes 53 and beyond.

My local installation on laravel valet does the same.

Any ideas?

**Comment by karandatwani92** [comment]
Hey @semiplay 

I just tried this case using `repeatable` columns, and I was able to store more than 52 items without any issues. In fact i limited it to 60, and it was working as expected. In DB my column type was `json`.

```php
[
    'name'      => 'features',
    'label'     => 'Features',
    'type'      => 'repeatable',
    'max_rows'     => 60,
    'subfields' => [
        [
            'name'    => 'feature',
            'wrapper' => [
                'class' => 'col-md-3',
            ],
        ],
        [
            'name'    => 'value',
            'wrapper' => [
                'class' => 'col-md-6',
            ],
        ],
        [
            'name'    => 'quantity',
            'type'    => 'number',
            'wrapper' => [
                'class' => 'col-md-3',
            ],
        ],
    ],
];
```

![Screenshot 2024-07-01 at 3 37 25 PM](https://github.com/Laravel-Backpack/community-forum/assets/8214221/2d864d3a-c11f-4bc2-ad44-bf7df5ef5597)

**Comment by semiplay** [comment, marked as answer]
I guess the difference in our setups is the volume of info stored in my model.

For anyone else who might find it useful, the problem turned out to be server resources.  Specifically the php.ini settings of **max_multipart_body_parts** and **max_input_vars**.  Once I increased those, it worked.
